src/Apps/Collect/CollectApp.tsx

```diff
@@ -12,14 +20,32 @@ export interface CollectAppProps {
 export class CollectApp extends Component<CollectAppProps> {
   render() {
     return (
-      <Flex flexDirection="column">
-        <Box my={2}>
-          <Serif size="8">Collect Art &amp; Design Online</Serif>
-        </Box>
-        <Box>
-          <ArtworkGrid viewer={this.props.viewer} />
-        </Box>
-      </Flex>
+      <>
+        <Title>Collect | Artsy</Title>
+        <Flex flexDirection="column">
+          <Box mt={3} mb={4}>
+            <Serif size="8">Collect Art &amp; Design Online</Serif>
+          </Box>
+          <Box>
+            <ArtworkGrid viewer={this.props.viewer} />
+          </Box>
+
+          {typeof window !== "undefined" && (
+            <>
```

Comment:
What is this?

Reply:
It's syntactic sugar for `<React.Fragment>` https://reactjs.org/docs/fragments.html

Reply:
Nice!